chroot-bin/croutonxinitrc-wrapper

```diff
@@ -126,6 +126,15 @@ if [ "$xmethodtype" != "xiwi" ]; then
     fi
 fi
 
+# Apply the Chromebook keyboard map in xiwi if using Freon. 
+if [ "$xmethodtype" = 'xiwi' ] && [ ! -f "/sys/class/tty/tty0/active" ]; then
```

Comment:
That would indeed be better. It's now factored out under the the condition `(!xiwi) || freon`.